bigquery_etl/shredder/delete.py

```diff
@@ -188,7 +189,7 @@ def get_task_id(target, partition_id):
     def create_job(client):
         query = dedent(
             f"""
-            DELETE
+            {"SELECT * FROM" if dry_run else "DELETE"}
```

Comment:
dry_run fails with `User does not have bigquery.tables.updateData permission for table` with a `DELETE` statement, but works and returns the correct total bytes processed with read-only permission using a `SELECT * FROM`